File: Assignment4/Assignment4/problem3.c
//Accept numbers from user and display its non factors
//Number -12
// Output - 6 4 3 2 1


#include<stdio.h>

int MultiFactor(int iNo)
{
  int iCnt =0;

  
  if(iNo<0)
  {
  	iNo=-iNo;
	 	
  }
 	
  
  for(iCnt=iNo;iCnt>=1;iCnt--)
  {   
	if((iNo%iCnt)!=0)
  	{   
  	    printf("%d\n",iCnt);
  	}
  }   

}

int main()
{
  int iValue =0;
  int iRet =0;
 
  printf("Enter Number\n");
  scanf("%d",&iValue);
   
  MultiFactor(iValue);
   


  return 0;
}

File: Assignment 5/Problem3.java
// Accept number from user and display its Non factors 

import java.lang.*;
import java.util.*;

class Maths
{
	public int iValue=0;
	
	public Maths(int iNo)
	{
		this.iValue=iNo;
	}
	
	public void NonFactor()
	{
		int iCnt =0;
		
		for(iCnt=1;iCnt<iValue;iCnt++)
		{
			if(iValue%iCnt != 0)
			{
				
				System.out.println(iCnt);
				
			}
			
		}
		
	}
	
}

class Assignment3
{
	
	public static void main(String arr[])
	{
		int iNumber =0;
		
		System.out.println("Enter Number");
		
		Scanner sobj = new Scanner(System.in);
		iNumber = sobj.nextInt();
		
		Maths mobj = new Maths(iNumber);
		
		mobj.NonFactor();
	}
	
}